src/transformers/models/bert/convert_bert_pytorch_checkpoint_to_original_tf.py

```diff
@@ -81,7 +81,7 @@ def create_tf_var(tensor: np.ndarray, name: str, session: tf.Session):
             if any(x in var_name for x in tensors_to_transpose):
                 torch_tensor = torch_tensor.T
             tf_var = create_tf_var(tensor=torch_tensor, name=tf_name, session=session)
-            tf.keras.backend.set_value(tf_var, torch_tensor)
+            tf_var.assign(torch_tensor)
```

Comment:
Is `.assign` backwards compatible?

Reply:
Yes - `.assign()` is a TF `Tensor` method that's been there since 2.0.

Reply:
The only difference is that `.assign()` doesn't cast different dtypes together automatically. The dtypes should always be `float32` here, but I'll add a cast for safety (also this code is ancient and looks like TF1, so I'm not sure if it runs at all on modern TF anyway)